stdlib/Profile/src/Profile.jl

```diff
@@ -63,6 +63,25 @@ macro profile(ex)
     end
 end
 
+export @profile_tasks
+
+"""
+    @profile_tasks
+
+`@profile_tasks <expression>` runs your expression while taking periodic backtraces of a sample of all live tasks (both running and not running).
+These are appended to an internal buffer of backtraces.
```

Comment:
Need to document (somewhere, maybe here, maybe in the manual), that this can be configured via `init`, same as the `@profile`, and that you can't use `@profile` at the same time as this.